src/main/java/com/mael/payroll/cards/TimeCard.java

```diff
@@ -0,0 +1,19 @@
+package com.mael.payroll.cards;
+
+public class TimeCard {
+    private int date;
+    private double hours;
+
+    public TimeCard(int date, double hours) {
```

Comment:
I'm curious, adding an `int` first and then having the need to change it later. Does that really simplifies or speed up anything? What would have been harder if a `Date` (or Java 8's new `LocalDate`) would have been used?